test/test_distributions.py

```diff
@@ -257,6 +263,14 @@ def test_log_prob(jax_dist, sp_dist, params, prepend_shape, jit):
     samples = jax_dist.sample(key=rng, size=prepend_shape)
     assert jax_dist.log_prob(samples).shape == prepend_shape + jax_dist.batch_shape
     if not sp_dist:
+        if isinstance(jax_dist, dist.TruncatedCauchy) or isinstance(jax_dist, dist.TruncatedNormal):
```

Comment:
If we add this to `_DIST_MAP`, we shouldn't have to add this specially.

Reply:
Unfortunately, there is no trunccauchy in scipy. In addition, scipy truncnorm does not support ndarray `low` arg. If I add `truncnorm`
```
sp_dist = osp_stats.truncnorm(a=onp.zeros(2), b=onp.ones(2), loc=onp.zeros(2), scale=onp.ones(2))
x = sp_dist.rvs(size=(1000, 2))
sp_dist.logpdf(x)  # scipy bug: ValueError
```
If we add `truncnorm` to DIST_MAP, then we can only test for scalars `low`, `loc`, and `scale` (because scipy `a` is `(low - loc) / scale`). I think that there is just a missing point here: missing the mean_var test w.r.t. scipy truncnorm implementation (with scalar parameters). We can add a separate test for it instead if you want.

Reply:
I see, I didn't realize that `truncnorm` wasn't vectorized w.r.t. `low`, and there is no truncated cauchy at all. it is probably not worth adding a separate test for that.